.github/workflows/main.yml

```diff
@@ -5,8 +5,7 @@ on:
       - 'master'
       - 'develop'
   pull_request:
-    branches:
-      - '*'
```

Comment:
@ActiveChooN , what did you make the change?

Reply:
@dvkruchinin  is the original author. Dmitry, can you comment, please?

Reply:
Without these changes, tests are not run if a PR is created for another PR. I added this here to make it clear that the adapted tests are successful for this PR. These changes can also be made by a separate PR.